.travis.yml

```diff
@@ -139,44 +143,47 @@ jobs:
         - docker exec docker_arm_emulator /bin/bash -c "cd /build && make install"
       before_script:
       after_failure:
+      after_success:
```

Comment:
why do we need this change instead of just setting CODECOV_FLAGS="-F arm"

Reply:
i didnt have env variable initially so this was leftover, removed